config/mainnetAddressScript.ts

```diff
@@ -21,13 +21,15 @@ export let addresses: Addresses = {
   ensPublicResolver: '{{mainnet.IPublicResolver}}',
   blockNumber: '',
   network: '',
+  tokenLockManager: ''
 }
 
 const main = (): void => {
   try {
     let output = JSON.parse(mustache.render(JSON.stringify(addresses), renameAddresses))
     output.blockNumber = '11440000' // Hardcoded a few thousand blocks before 1st contract deployed
     output.network = 'mainnet'
+    output.tokenLockManager = "0xFCf78AC094288D7200cfdB367A8CD07108dFa128"
```

Comment:
Do we need to do this because the addresses aren't configured in the package?

Reply:
exactly. there are not part of the core contracts, and thus are not in the package!